test/SavedSearches/UDB3SavedSearchRepositoryTest.php

```diff
@@ -0,0 +1,194 @@
+<?php
+
+namespace CultuurNet\UDB3\SavedSearches;
+
+use Broadway\UuidGenerator\UuidGeneratorInterface;
+use CultuurNet\UDB3\DBALTestConnectionTrait;
+use CultuurNet\UDB3\SavedSearches\Doctrine\SchemaConfigurator;
+use CultuurNet\UDB3\SavedSearches\Properties\QueryString;
+use CultuurNet\UDB3\SavedSearches\ReadModel\SavedSearch;
+use ValueObjects\StringLiteral\StringLiteral;
+
+class UDB3SavedSearchRepositoryTest extends \PHPUnit_Framework_TestCase
+{
+    use DBALTestConnectionTrait;
+
+    /**
+     * @var StringLiteral
+     */
+    private $tableName;
+
+    /**
+     * @var UuidGeneratorInterface|\PHPUnit_Framework_MockObject_MockObject
+     */
+    private $uuidGenerator;
+
+    /**
+     * @var UDB3SavedSearchRepository
+     */
+    private $udb3SavedSearchRepository;
+
+    protected function setUp(): void
+    {
+        $this->createTable();
+
+        $this->uuidGenerator = $this->createMock(UuidGeneratorInterface::class);
+
+        $this->udb3SavedSearchRepository = new UDB3SavedSearchRepository(
+            $this->getConnection(),
+            $this->tableName,
+            $this->uuidGenerator,
+            new StringLiteral('6f072ba8-c510-40ac-b387-51f582650e26')
+        );
+    }
+
+    /**
+     * @test
+     * @throws \Doctrine\DBAL\DBALException
```

Comment:
throws hier weglaten, is niet relevant in deze unit test, enkel verwarrend dat die hier vermeld staat (vs. expectedException)